verma2021explaining/openreview/packages.tex

```diff
@@ -0,0 +1,34 @@
+\usepackage{neurips_2019}
```

Comment:
Please remove this import, as we use ReScience style files. This is why the compilation is failing.

Reply:
Hi, Removed the import. But make the issue still persists. Kindly refer to the https://gist.github.com/aritzzz/7c8e96b73171990a2f353fc3298ec304 to see the article.log. To summarise: We do get [article.pdf(](https://github.com/aritzzz/RC-2020/blob/main/verma2021explaining/journal/article.pdf) but it does not render any images, neither bibliography and appendix.